lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.html

```diff
@@ -1,49 +0,0 @@
-<form [formGroup]="form" novalidate (ngSubmit)="apply(form.value, form.valid)">
-    <mat-form-field>
-        <mat-label>{{ 'SEARCH.FILTER.RANGE.SELECT-FROM-DATE' | translate }}</mat-label>
-        <input matInput
-                (input)="forcePlaceholder($event)"
-                [formControl]="from"
-                [errorStateMatcher]="matcher"
-                placeholder="{{ 'SEARCH.FILTER.RANGE.FROM-DATE' | translate }}"
-                [attr.aria-label]="'SEARCH.FILTER.RANGE.FROM-DATE' | translate"
-                [matDatepicker]="fromDatepicker"
-                [max]="fromMaxDate"
-                (dateChange)="onChangedHandler($event, from)"
-                data-automation-id="date-range-from-input">
-        <mat-datepicker-toggle matSuffix [for]="fromDatepicker" data-automation-id="date-range-from-date-toggle"></mat-datepicker-toggle>
```

Comment:
this variable seems still referencing to that removed element, please remove that variable as this element does not exist anymore: 
<img width="945" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/21e485d4-8e29-4250-a8ae-fda1b53f0b44">

Reply:
The only use for the date-range-filter.page.ts file was in search-categories page. Once that reference was [removed](https://github.com/Alfresco/alfresco-ng2-components/pull/9023#discussion_r1374624065), the file was no longer needed. 
It has now been removed